app/js/containers/ArtifactsList.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import Artifacts from '../components/Artifacts';
+import * as actionCreators from '../actions/Artifacts';
```

Comment:
I forgot to ask about that. Do we want to pull in the entirety of the action creators list (not really sure if it affects anything either way performance wise)? Then it should be that way for sure, but this was just pulling in the artifact specific actions.